Python/bltinmodule.c

```diff
@@ -2010,16 +2010,16 @@ builtin_input_impl(PyObject *module, PyObject *prompt)
 
         /* stdin is a text stream, so it must have an encoding. */
         stdin_encoding = _PyObject_GetAttrId(fin, &PyId_encoding);
+        if (!stdin_encoding || !PyUnicode_Check(stdin_encoding))
```

Comment:
This should be
```c
if (!stdin_encoding || !PyUnicode_Check(stdin_encoding)) {
    goto _readline_disable_tty_and_fall_back;
}
```
as it is specified in PEP 7 (https://www.python.org/dev/peps/pep-0007/#code-lay-out):

>  All new C code requires braces